app/gui/src/controller/searcher.rs

```diff
@@ -824,7 +850,7 @@ impl Searcher {
             self.mode.node_id(),
             Box::new(|md| md.intended_method = intended_method),
         )?;
-        tracing::debug!("Previewing expression: \"{:?}\".", expression);
+        tracing::warn!("Previewing expression: \"{:?}\".", expression);
```

Comment:
use `warn!` instead of `tracing::warn!`. Linter should warn about unnecessary qualification here